src/test/java/seedu/programmer/logic/LogicManagerTest.java

```diff
@@ -52,7 +52,7 @@ public void setUp() {
 
     @Test
     public void execute_invalidCommandFormat_throwsParseException() {
-        String invalidCommand = "uicfhmowqewca";
+        String invalidCommand = "inavlid command";
```

Comment:
Is this a typo here or intentional?

Reply:
Intentional. "invalid command" is still an invalid command, and provides more info to the developer reading

Reply:
okay, makes sense!